5.0/en/0x18-V10-Malicious.md

```diff
@@ -54,9 +54,12 @@ Complying with this section is likely to be operational and continuous.
 | :---: | :--- | :---: | :---: | :---: | :---: |
 | **10.4.1** | [ADDED] Verify that the application explicitly ensures that variables are of the correct type and performs strict equality and comparator operations to avoid type juggling or confusion vulnerabilities caused by the application code making an assumption about a variable type. | ✓ | ✓ | ✓ | 843 |
 | **10.4.2** | [ADDED] Verify that the application avoids DOM clobbering when using client-side JavaScript by employing explicit variable declarations, performing strict type checking, avoiding storing global variables on the document object, and implementing namespace isolation. | | ✓ | ✓ | 79 |
+| **10.4.3** | [ADDED] Verify that JavaScript code is written in a way that prevents prototype pollution by using Set() or Map() instead of object literals or, if this is not possible, by using some other protection mechanism. | | ✓ | ✓ | |
```

Comment:
The last part is not-actionable noise.

Maybe:
_Verify that JavaScript code is written in a way that prevents prototype pollution, e.g. by using Set() or Map() instead of object literals._

Reply:
There are other mechanisms which are detailed in the cheat sheet which is one of the reasons I added a link to it.